xcode/Subconscious/Shared/Services/DatabaseService.swift

```diff
@@ -1130,7 +1168,7 @@ final class DatabaseService {
         
         return try? database.execute(
             sql: """
-            SELECT id, modified, excerpt
+            SELECT did, id, modified, excerpt
```

Comment:
I would like to update/add a test for readRandomEntryMatching to ensure we're getting the data out correctly.

Reply:
Added `Tests_DatabaseService.testReadRandomEntryMatching`